pyEpiabm/pyEpiabm/sweep/initial_household_sweep.py

```diff
@@ -0,0 +1,440 @@
+#
+# Sweep to assign people to households and set their ages
+#
+
+import random
+import numpy as np
+
+from pyEpiabm.core import Parameters, Person, Population
+from .abstract_sweep import AbstractSweep
+
+
+class InitialHouseholdSweep(AbstractSweep):
+    """Class to assign ages to people in
+    households using the same age distribution
+    model as CovidSim.
+    """
+
+    def __init__(self):
+        """Call in variables from the parameters file.
+        """
+        # worth putting all household params into one and changing name to just
+        # household_params?
+        self.use_ages = Parameters.instance().use_ages
+        self.household_size_distribution \
+            = Parameters.instance().household_size_distribution
+        self.max_household_size = Parameters.instance().max_household_size
+        self.age_params = Parameters.instance().household_age_params
+        self.age_proportions = Parameters.instance().age_proportions
+        self.num_age_groups = len(self.age_proportions)
+        self.age_group_width = 5
+
+    def household_allocation(self, population: Population):
+        """Method that takes in a population and assigns them to different
+        sized hosueholds according to a household size distribution.
+
+        Parameters
+        ----------
+        Population: Population
+            Instance of Population class
+        """
+
+        for cell in population.cells:
+            for microcell in cell.microcells:
+                k = 0
```

Comment:
```suggestion
                k = 0  # Counter of people with allocated household in mcell
```

May need to split over another line if it fails style test